php/class-wp-js-widget.php

```diff
@@ -100,7 +102,7 @@ public function enqueue_frontend_scripts() {}
 	 * @return array Schema.
 	 */
 	public function get_item_schema() {
-		$schema = array(
+		$item_schema = array(
 			'title' => array(
 				'description' => __( 'The title for the widget.', 'js-widgets' ),
 				'type' => array( 'string', 'object' ),
```

Comment:
@westonruter I've noticed such 'compound' field types here and there. I'm not sure how they are meant to be used, but I've noticed in `get_default_instance()` method, around line `150` such condition:

```
if ( 'object' === $options['type'] ) {
```

I think if would fail, since `object` is just one item inside an array here. You might want to review it.